For usbredirfilter_string_to_rules()
--- spice-gtk-0.9/configure.ac.orig	2012-02-04 08:31:07.000000000 +0100
+++ spice-gtk-0.9/configure.ac	2012-02-04 12:03:50.651039985 +0100
@@ -344,7 +344,7 @@
   AM_CONDITIONAL(WITH_USBREDIR, false)
 else
   PKG_CHECK_MODULES([USBREDIR],
-                    [gudev-1.0 libusb-1.0 >= 1.0.9 libusbredirhost >= 0.3.3],
+                    [gudev-1.0 libusb-1.0 >= 1.0.9 libusbredirhost >= 0.3.3 libusbredirparser >= 0.3.3],
                     [have_usbredir=yes],
                     [have_usbredir=no])
   if test "x$have_usbredir" = "xno" && test "x$enable_usbredir" = "xyes"; then
