--- spice-gtk-0.29/src/Makefile.am.orig	2012-01-31 14:31:38.000000000 +0100
+++ spice-gtk-0.29/src/Makefile.am	2012-02-04 08:30:40.437278387 +0100
@@ -549,7 +549,7 @@
 DEFSDIR = `pkg-config --variable=defsdir pygtk-2.0`
 
 spice-client-gtk.defs: $(libspice_client_gtkinclude_HEADERS) $(nodist_libspice_client_gtkinclude_HEADERS) $(libspice_client_glibinclude_HEADERS) $(nodist_libspice_client_glibinclude_HEADERS)
-	$(AM_V_GEN)$(PYTHON) $(CODEGENDIR)/h2def.py \
+	$(AM_V_GEN)$(PYTHON) $(CODEGENDIR)/h2def.pyc \
 		-f $(srcdir)/spice-client-gtk-manual.defs \
 		$^ > $@
 
