--- spice-gtk-0.30/spice-common/common/Makefile.am.orig	2015-09-03 17:25:34.000000000 +0200
+++ spice-gtk-0.30/spice-common/common/Makefile.am	2015-10-02 19:20:13.650132689 +0200
@@ -100,12 +100,6 @@
 	$(NULL)
 
 MARSHALLERS_DEPS =							\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/__init__.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/codegen.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/demarshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/marshal.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/ptypes.py		\
-	$(CODE_GENERATOR_BASEDIR)/python_modules/spice_parser.py	\
 	$(CODE_GENERATOR_BASEDIR)/spice_codegen.py			\
 	$(NULL)
 
